docs/resources/aws_security_group_firewall_rules_v2.md

```diff
@@ -0,0 +1,67 @@
+---
+page_title: "instaclustr_aws_security_group_firewall_rules_v2 Resource - terraform-provider-instaclustr"
+subcategory: ""
+description: |-
+---
+
+# instaclustr_aws_security_group_firewall_rules_v2 (Resource)
+A listable data source of all AWS Security Group based firewall rules in an Instaclustr managed cluster.
```

Comment:
this is a resource not a datasource?

Reply:
The GET request(instaclustr_aws_security_group_firewall_rules_v2_instance) would be its datasource